app/helpers/editions_helper.rb

```diff
@@ -19,11 +19,28 @@ def legacy_resource_form(resource, &form_definition)
     nested_form_for resource, as: :edition, url: edition_path(resource), html: html_options, &form_definition
   end
 
-  def format_conversion_select_options(edition)
+  def legacy_format_conversion_select_options(edition)
     possible_target_formats = Edition.convertible_formats - [edition.artefact.kind]
     possible_target_formats.map { |format_name| [format_name.humanize, format_name] }
   end
 
+  def format_conversion_radio_options(edition)
```

Comment:
Rather than have these two methods (`format_conversion_radio_options` and `radio_options_hints`), consider having just the one that returns the list of items (i.e. an array `[{value: "", text: "", hint_text: ""}…]`).

Reply:
Right, that would be more elegant. I think I have understood and updated accordingly.